subsys/net/l2/openthread/openthread.c

```diff
@@ -375,14 +382,15 @@ int openthread_start(struct openthread_context *ot_context)
 		otThreadSetExtendedPanId(ot_instance, &xpanid);
 	}
 
-	NET_INFO("OpenThread version: %s", otGetVersionString());
 	NET_INFO("Network name: %s",
 		 log_strdup(otThreadGetNetworkName(ot_instance)));
 
 	/* Start the network. */
-	error = otThreadSetEnabled(ot_instance, true);
-	if (error != OT_ERROR_NONE) {
-		NET_ERR("Failed to start the OpenThread network [%d]", error);
+	if (!IS_ENABLED(CONFIG_OPENTHREAD_COPROCESSOR)) {
```

Comment:
This change does not seem to be needed (given that early exit at line 348 is done for co-processor in general). I've tested it, no build errors.

Reply:
Done.